core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.ts

```diff
@@ -38,13 +38,27 @@ export class SideNavigationBarComponent {
   // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
   @Input() display!: boolean;
 
-  IMPACT_REPORT_LINK = AppConstants.IMPACT_REPORT_LINK;
+  impactReports = [
+    {
+      link: AppConstants.IMPACT_REPORT_LINK_2023,
+      text: '2023',
+      className:
+        'oppia-impact-report-button oppia-sidebar-menu-link e2e-mobile-test-sidebar-impact-report-2023-button',
```

Comment:
why do you need separate classes for 2023 and 2022? Shouldn't these be uniform and declared in the html page itself?